build/hellobuild.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Hello Build
+description: Build Hello World
+keywords: build, buildx, buildkit, getting started, Dockerfile, image layers, build instructions, build context
+---
+
+## Hello Build!
+
+It all starts with a Dockerfile.
+
+Dockerfiles are text files containing instructions. Dockerfiles adhere to a specific format and contain a **set of instructions** for which you can find a full reference in the [Dockerfile reference](../engine/reference/builder).
+<br/>Docker builds images by reading the instructions from a Dockerfile.
+
+Docker images consist of **read-only layers**, each resulting from an instruction in the Dockerfile. Layers are stacked sequentially and each one is a delta representing the changes applied to the previous layer.
+
+## Dockerfile basics
+
+A Dockerfile is a text file containing all necessary instructions needed to assemble and deploy a container image with Docker.
```

Comment:
```suggestion
A Dockerfile is a text file containing all necessary instructions needed to assemble your application into a Docker container image.
```

I'd leave off the `deploy` part, deployment isn't really part of the concern for build, more for runtime.